### DISCUSSION TITLE
定时执行任务能否非整点

### DISCUSSION THREAD

**Discussion by CCCClaus** [MaaAssistantArknights/MaaAssistantArknights#3215] [Ideas]
话说定时执行为什么只能是整点呢？
就比如类似3：50或者3：40执行一次还是有一定的需求的。

**Comment by MistEO** [comment]
只是因为懒得写（

**Reply by MistEO** [reply]
欢迎各位大佬 pr

**Comment by ABA2396** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/50f27516d36d246a0a0ebdc9fb2177136bc06058 completed
